src/brain/gocdSlackFeeds/deployFeed.ts

```diff
@@ -298,7 +298,12 @@ export class DeployFeed {
   }
 
   async getBodyText(pipeline: GoCDPipeline) {
-    let body = `GoCD deployment started`;
+    let body  = '';
+    if (pipeline.stage.result.toLowerCase() == 'failed') {
```

Comment:
It seems that when one of these deploy messages is posted to slack it starts off as unknown then eventually gets to passed or failed. So I'm concerned that when this function is triggered it will have result unknown, later change to failed, but when it changes to failed not trigger the function again, thus the fail message never gets posted....